iOS/Router/MoviesRouterNavigation.swift

```diff
@@ -0,0 +1,27 @@
+import Shared
+
+class MoviesRouterNavigation: MoviesRouter {
+
+    private var navigationController: UINavigationController
+
+    init(navigationController: UINavigationController) {
+        self.navigationController = navigationController
+    }
+
+    func root() {
+        let upcomingMoviesViewController = UpcomingMoviesViewController()
+        addTitleInNavigation(at: upcomingMoviesViewController, with: "UPCOMING MOVIES")
+        navigationController.viewControllers = [upcomingMoviesViewController]
+    }
+
+    func movieDetail(movie: Movie, movieGenres: [Genre], movieBackdropImage: UIImage?) {
+        let movieDetailViewController = MovieDetailViewController(
+            movie: movie, movieGenres: movieGenres, movieBackdropImage: movieBackdropImage)
+        navigationController.pushViewController(movieDetailViewController, animated: true)
+    }
+
+    private func addTitleInNavigation(at viewController: UIViewController, with text: String) {
```

Comment:
Yes. It is in my dev-belt-snippets with more lines. But I just drop the extra lines and forgot to refactor :/